jumanji/wrappers.py

```diff
@@ -215,6 +215,22 @@ def action_spec(self) -> dm_env.specs.Array:
     def unwrapped(self) -> Environment:
         return self._env
 
+    @property
+    def state(self) -> Any:
```

Comment:
Could we use `State` instead of `Any` here?
```suggestion
    def state(self) -> State:
```
Would you mind applying the same to `self._state` in the `__init__`?